Source/WTF/wtf/URL.h

```diff
@@ -212,6 +212,9 @@ class URL {
     unsigned credentialsEnd() const;
     void remove(unsigned start, unsigned length);
     void parse(String&&);
+    void parseAllowingC0AtEnd(String&&);
+
+    void potentiallyStripTrailingSpacesFromOpaquePath();
```

Comment:
We usually use "maybe" instead of "potentially" for things like this.

Reply:
Thanks! Also renamed "Strip" to "Trim" in line with similar functions.